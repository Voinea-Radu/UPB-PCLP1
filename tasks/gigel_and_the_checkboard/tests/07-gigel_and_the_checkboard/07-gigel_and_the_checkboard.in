72
259 629 -262 -801 -397 -675 552 -95 953 -9 -362 -355 432 -222 -565 -152 342 -409 -227 -525 -987 30 -452 430 -888 -563 -988 772 -378 -962 958 -74 783 -323 -138 643 -1004 -1017 -485 -184 0 -937 -829 524 766 291 354 641 -580 -986 778 -31 -795 557 768 -418 969 -280 -170 -595 -883 -875 -188 -791 924 -282 791 -927 -743 -816 624 -337 
-831 903 -562 -422 374 -823 -202 678 670 629 -194 -407 251 -158 -358 964 -18 -197 -708 -152 812 -682 6 -759 483 269 -662 6 1 429 -677 278 239 -432 1018 -137 -385 -57 908 546 635 -70 619 -733 -878 -574 183 237 1007 -388 -990 -387 675 -551 912 -35 859 -442 -925 -945 -879 -400 799 201 -547 902 820 938 -426 -430 -149 -525 
-179 630 -952 268 232 83 418 53 725 655 -104 -859 440 -634 1012 1004 -189 -867 -759 79 -17 -589 -1023 944 793 -957 700 386 -237 -493 274 -170 -72 -646 357 63 -250 773 -993 965 -819 473 449 590 -425 129 -704 306 -787 741 -590 -687 -429 851 610 93 131 973 -602 -723 572 -179 154 677 432 -469 76 627 -894 249 -238 38 
-266 -841 -98 -878 -595 -255 -84 -614 90 878 -501 -89 430 7 -443 -364 -908 864 -65 499 279 904 -482 -474 -955 -507 -154 61 -21 374 88 456 -927 -133 -672 -464 -153 610 -7 -831 326 37 88 787 299 536 -1014 -519 171 -874 -872 -263 153 -39 55 -592 778 231 202 1011 -44 551 477 715 32 -984 448 327 -517 -509 398 142 
-233 362 672 -392 184 -77 343 -752 -741 89 739 -1022 -247 211 894 95 -317 486 -104 -1004 750 694 976 763 -537 210 36 909 -6 455 -684 -556 902 -779 213 -511 -711 55 241 -936 774 -600 -929 231 -150 -52 -489 264 45 -3 812 -6 804 371 364 -289 -470 268 997 765 719 362 583 -991 502 -52 -969 -595 -61 75 -969 -663 
189 -1012 578 69 663 -564 732 -709 -427 56 -938 -354 417 -913 -220 729 -87 0 627 682 -58 -135 482 497 605 -371 -760 -128 946 -267 -642 -363 -775 948 -88 123 -557 -698 123 67 -532 244 510 529 -191 -633 932 635 -131 622 -70 828 -555 497 -956 -15 -161 -463 -551 437 856 -566 605 611 -1004 186 -148 -548 -906 460 912 -1020 
-19 507 -622 471 -868 -564 -523 54 254 204 -992 632 79 -38 379 611 906 931 -967 -516 -700 604 421 -740 478 611 -500 676 -270 -110 -631 434 670 -426 -245 -900 -356 -264 268 -651 15 -760 546 -423 -176 544 -165 -410 143 325 -677 -710 -57 317 -303 774 -939 517 -802 682 -475 -983 -771 -400 317 -1024 -95 -545 -742 -221 35 331 
709 639 -1013 691 25 -778 -954 828 -1016 -691 -886 -342 -71 485 -18 -555 787 376 -653 -654 824 -127 -54 663 697 551 344 -180 -140 -698 -500 -149 -778 150 -502 307 874 911 359 -1022 -587 -829 -927 -664 597 -579 65 706 -518 865 80 -328 -679 800 -798 -446 122 -20 -629 412 -46 -485 455 285 -743 167 -301 -344 586 -632 778 404 
0 413 760 886 913 -48 -952 -71 84 982 1003 824 -998 453 -674 595 510 -987 274 908 700 -483 -199 298 -8 650 708 -877 -170 -275 0 -427 655 714 -582 945 -349 -935 959 -1002 -867 522 275 73 -805 -860 -634 503 -211 -880 -584 -367 614 39 315 197 -305 -692 938 -437 -514 535 -904 577 -299 524 -591 864 520 -552 -248 -720 
-365 794 -621 -127 -110 -657 -1022 -562 -223 165 785 17 -477 704 284 -210 675 -240 20 316 595 998 -400 -36 -360 564 -883 -732 -927 119 -764 -161 -17 275 -992 654 612 -66 -653 -786 373 297 -839 181 -111 -74 408 164 721 633 -898 325 -417 10 547 81 117 -238 -70 140 58 527 -827 -746 -482 -833 -561 411 -541 313 -910 -685 
32 887 678 -772 54 -814 -723 -840 -830 -582 420 705 779 -690 -745 383 -821 213 771 382 193 -869 219 -971 140 198 -1021 773 -182 958 431 -13 -446 -745 -80 -814 82 969 46 781 414 -655 -762 962 -441 418 796 -546 977 -961 -105 -502 -311 -161 655 532 435 843 -468 121 851 -632 -484 -632 -910 986 -161 -104 248 2 758 421 
-377 860 -366 639 -146 -705 -970 533 176 -517 -981 786 657 565 305 299 -189 -608 217 661 873 -96 872 445 673 590 370 222 -233 373 -47 516 -667 -76 317 285 22 -726 -130 658 -83 685 402 334 -880 -842 260 -310 690 -757 134 -890 -27 560 -201 -980 324 923 -750 942 140 -666 -179 -698 -159 -189 -309 -735 -293 -931 -558 -849 
-462 -424 -729 1018 -915 -727 486 291 336 -1007 108 733 -341 -580 -946 -452 -439 268 -723 -50 -859 -565 -105 -32 275 -232 775 -629 664 204 -649 -452 385 731 -105 -367 122 -525 496 -41 -164 -489 969 841 -498 258 446 -116 -596 708 886 -683 471 655 -706 -852 -669 625 36 334 19 -446 292 495 -38 217 -428 750 -418 -775 494 -1024 
-122 -96 791 310 584 917 -120 788 1009 -245 126 134 -776 843 -381 -578 191 242 -279 -286 268 948 -418 -432 -937 -187 -882 -400 151 -610 995 -959 -419 -943 274 872 -612 605 675 -806 -142 627 872 -1017 499 -595 858 276 -789 -440 390 -1018 -293 -613 -203 945 -67 649 -391 209 -58 398 53 363 -732 352 992 82 242 -552 656 617 
582 -949 -229 642 -992 1024 493 -846 657 67 -496 3 -1008 -474 154 839 292 -997 503 -199 133 -268 -243 -607 1024 -656 -278 -785 79 219 -198 -380 198 560 -290 -458 201 -1005 955 719 815 -163 297 95 592 661 751 1015 -247 765 -925 437 647 517 892 -435 242 316 90 631 -904 481 75 725 -306 292 -551 -127 776 648 -141 -444 
-340 539 331 -98 781 631 -180 -263 231 131 -162 707 858 -337 -353 -998 -675 727 208 705 118 927 -30 -911 790 -265 154 -854 689 442 532 181 -159 -861 560 829 -938 -135 96 -600 346 914 585 -778 -453 -432 -704 554 -645 -5 -604 291 466 -344 -651 -847 -329 605 -339 525 866 26 90 -337 799 88 595 -15 -236 133 128 -181 
396 -989 691 585 -1006 61 486 179 -293 -397 -429 -196 -807 -576 -785 -585 44 -812 858 0 1012 -255 -937 310 -618 -466 55 -720 -575 -156 -135 1020 176 126 -703 85 222 173 -672 94 -705 -445 -173 764 316 779 -92 -222 708 980 620 366 -166 272 -717 -596 309 -603 -289 -525 32 458 932 472 832 -338 -346 44 -764 823 362 -295 
298 -693 795 236 282 -526 -456 -883 -627 -442 -897 -76 -339 738 -930 826 -976 108 -126 -245 -249 -24 160 118 -739 802 -926 -968 -595 -470 -202 687 45 -687 -17 821 980 21 -341 594 210 427 870 -460 822 -284 -604 504 927 -248 -626 276 -601 179 60 365 624 945 -219 -277 1007 24 428 680 17 381 354 -544 888 496 285 -234 
992 704 -139 145 -187 340 -499 814 681 322 -816 -1022 -582 -113 55 -628 309 -816 -16 -405 804 -583 251 559 893 -1007 -893 -48 -276 -765 690 -977 -592 11 -654 -890 -591 471 333 -359 607 270 -445 106 -997 -440 -99 -913 74 -685 -653 -86 641 150 -936 -824 -845 -107 561 482 -434 -850 681 243 334 739 971 470 768 342 765 809 
-1008 -830 647 902 619 -354 463 -970 497 -538 -1021 520 482 -1022 -631 683 536 466 457 487 -863 -268 952 -85 180 519 489 776 -998 981 88 -127 -737 -250 242 -709 -987 -56 -265 63 379 110 886 416 -585 92 463 90 -273 -966 -448 -672 399 -827 -449 -264 96 700 642 715 339 307 -52 915 -90 565 595 -735 327 -306 -139 -471 
83 -1014 351 907 -339 705 333 -415 513 -918 642 164 -82 169 55 964 7 302 -655 -674 -891 563 -172 -489 970 -375 760 848 -984 -667 -286 -1022 799 922 -387 -478 757 173 -346 652 -899 436 -1001 -260 989 93 456 -727 463 309 -358 778 918 829 -466 -560 -509 755 410 -148 193 -938 -995 -426 244 866 -768 -724 630 -114 308 68 
-408 395 -863 555 26 962 840 100 -1014 157 471 -1022 765 123 -233 344 925 -252 370 66 962 -711 445 -288 -673 433 -831 1 633 -616 284 -325 -246 165 -241 -776 290 534 189 -59 478 75 -728 -176 -137 -954 -952 778 -234 647 -529 208 534 -352 -745 191 55 966 987 408 -369 -762 649 941 -1015 575 236 -24 -227 -297 -397 123 
141 458 -691 -316 -665 451 -855 -320 285 155 -851 53 -87 -316 -71 767 802 -393 458 670 -775 198 -634 -789 936 -589 -6 -533 -890 -345 246 -77 -574 141 213 55 -648 -73 750 -93 326 559 1020 460 988 -868 175 -939 657 13 807 730 -3 941 946 323 -521 -163 -421 -596 -615 583 -633 -982 751 870 57 -80 1 -461 76 -780 
845 179 92 712 -947 834 -1018 244 -1007 -155 -481 -680 -701 -508 -86 884 551 903 -699 -731 -187 -315 1000 -918 -960 549 -346 -240 -650 983 -791 428 -529 931 707 466 -837 -628 -380 54 339 1009 -729 -17 1009 -280 -346 248 -417 1002 817 289 -770 360 -327 423 -141 -119 -721 888 209 288 244 -973 -125 -320 -655 -285 728 -946 436 176 
574 -470 570 -775 609 355 -526 -828 -736 383 -306 772 7 939 323 973 977 146 -257 -488 196 917 -841 607 -908 82 169 504 -363 0 394 -876 -622 -756 396 -288 650 -79 804 40 589 16 -659 1019 -674 -228 -269 -184 584 697 -233 224 -510 -2 -225 12 -833 -930 676 -644 -912 761 554 921 602 453 595 564 844 68 315 -212 
-590 183 747 945 -792 649 89 -777 441 571 964 -374 -547 66 -279 453 399 216 -980 -709 752 -1004 672 -632 978 406 -332 975 -240 -598 789 515 -325 -351 -305 750 380 699 -876 198 441 585 855 -844 814 1008 -362 833 98 18 -921 537 -314 641 780 855 1000 -928 707 411 -305 75 796 -165 -570 666 -239 579 -141 -468 492 -160 
384 -585 628 323 -431 -304 650 930 782 205 695 275 501 -172 -539 -269 798 -60 -838 -534 -853 -1014 -591 148 -431 -984 -933 -817 -376 52 722 426 138 -952 901 932 654 653 -204 357 -822 695 675 164 -456 429 285 -509 55 -969 -517 -902 299 123 14 -746 696 488 478 629 422 -722 -100 35 133 768 106 -555 -618 917 -934 -483 
-881 -273 -382 -352 -427 106 442 231 398 723 -780 44 971 349 -31 -949 581 180 760 -544 669 -48 -692 -936 375 -305 -593 -498 433 449 -510 743 -100 -800 -576 478 941 -827 809 83 415 -668 272 -836 -582 374 -852 416 295 138 451 800 -527 204 768 -743 431 -511 176 -989 2 -871 117 -475 711 218 -397 -195 993 -129 -722 -263 
-694 -954 341 563 -449 -958 997 549 429 -88 -632 -31 225 -817 257 607 388 -715 -444 -402 -130 -488 516 -75 -877 257 851 -181 619 265 -339 -946 429 684 788 1016 -130 311 542 434 -799 130 -699 -166 -329 617 -878 497 -887 81 500 832 750 145 75 663 164 542 -869 846 -904 187 -982 -853 0 569 8 -486 -998 391 -389 394 
185 454 552 514 -583 -636 593 213 -393 250 -895 -442 195 -100 939 527 -948 919 165 -8 601 431 254 900 -435 -341 -561 -1009 253 223 435 137 -697 -778 683 556 206 -350 -354 800 -722 -791 589 -353 -960 -812 -781 -284 -26 4 22 995 385 279 -374 -466 336 -211 -349 -561 890 -314 -816 362 567 99 -163 972 -758 -922 592 -440 
-431 -530 1008 319 -666 350 -26 167 630 -507 -411 403 658 -188 451 292 -933 460 -120 -399 56 536 -341 -835 -104 266 -636 273 586 291 168 -828 202 -638 -980 -704 -623 151 169 -874 -97 -1023 649 -833 321 262 -934 -454 987 -856 -846 707 -572 929 -458 606 -261 -622 -446 133 228 -754 877 -60 629 13 -489 700 81 -460 -963 657 
-273 256 930 539 -970 975 45 -384 597 942 951 194 -442 313 782 1017 -941 564 311 756 513 80 204 214 -979 671 -906 -874 854 -426 -532 -839 -673 249 -251 -173 -691 211 805 100 -354 871 345 833 -215 496 -776 923 -672 -129 97 -536 -530 845 -380 578 701 769 206 -214 -958 -61 169 -799 326 -977 132 -832 311 649 851 956 
-955 -787 -796 -697 77 -94 596 -478 614 -381 335 -390 -889 931 114 402 459 -72 -454 -647 280 -786 2 -322 -99 -494 711 -747 -834 494 870 -953 924 -407 -548 -511 995 -881 943 710 -913 292 -414 604 398 98 1005 379 -305 -35 -713 -362 -984 -72 -628 -953 670 838 -324 323 -23 -884 -762 -334 305 -979 918 506 -575 -453 356 -677 
43 -824 307 851 391 -789 889 -155 987 344 506 208 -133 -291 -1009 -983 -909 185 620 -93 871 135 182 -646 631 -869 -598 242 820 35 954 -15 565 591 834 -817 1016 -523 981 971 -210 322 -924 1014 59 739 -162 395 -94 -406 -110 -506 880 -113 -837 382 -622 -508 360 -315 537 -984 -626 203 376 922 634 72 -982 -902 668 -47 
-1002 -95 535 813 998 -1021 -939 214 -694 282 843 168 -489 -550 698 -79 -192 279 790 800 469 605 -398 119 -454 -377 -286 740 826 167 513 561 272 992 -689 -415 -281 -981 214 753 -644 -184 264 1008 835 -968 314 368 -81 824 -469 923 -382 -168 -291 336 274 567 -401 288 -933 117 -175 -743 49 138 -159 -337 -917 -124 805 26 
516 -619 615 -347 -22 752 -906 121 700 -653 -612 -222 754 -603 -246 742 -491 105 -880 -948 693 -379 -889 -845 -571 -125 -166 499 85 986 480 -851 -993 541 -88 -32 -996 147 -388 -817 -935 516 -629 -1012 605 751 376 915 -81 362 -676 -196 -352 -535 238 -816 -27 223 -960 -35 620 897 602 -710 -643 -906 214 -452 -200 442 436 -317 
639 134 305 218 -319 -961 -523 -577 811 956 585 370 619 498 -6 -395 -979 -9 493 -61 -917 -939 -377 494 829 -590 -163 361 -327 -495 285 756 -138 211 -243 593 -527 753 -924 -756 117 738 -447 123 160 120 -916 -279 228 817 -286 457 -917 292 -21 -530 797 -362 -971 -985 1017 527 -995 655 952 217 956 40 34 522 65 -314 
-616 43 -60 109 508 894 -339 -360 249 -683 -601 -296 554 -943 -108 -413 193 611 -496 880 124 476 285 -79 -1022 -725 -709 -369 -492 211 455 460 -792 69 141 -245 -929 49 -429 838 496 959 645 912 -529 926 -217 -311 -309 935 398 -61 -541 -685 832 497 20 -632 315 -935 887 -189 -631 563 -429 601 -594 899 -316 -931 -881 424 
-996 -989 614 -306 820 -32 416 -877 732 287 -760 642 -252 286 -938 -51 -298 -861 668 -1009 812 518 -313 49 49 677 -823 -384 -536 308 587 -845 -686 -958 270 -172 875 443 -930 458 504 795 -814 304 -841 633 -683 135 647 -155 -450 -683 468 -369 425 -343 -424 218 -618 635 -492 -150 -230 -150 -147 -880 314 313 149 3 -722 309 
739 -111 215 406 956 704 624 44 647 1021 -785 -238 -966 141 136 -458 -796 -922 99 -972 -832 -711 -298 28 466 922 869 -809 -167 367 613 554 834 -258 -612 288 -885 744 981 -336 540 -943 1021 -489 -535 996 979 -300 -69 -178 -27 551 -314 -56 -352 590 172 648 1004 267 534 712 -936 75 948 -288 -106 10 74 240 -830 -158 
-978 410 406 -657 233 128 -956 939 210 669 -679 345 -571 -661 -504 142 161 -72 602 794 363 795 742 884 -169 477 -1013 733 -685 644 -1021 -239 -217 -702 290 -452 509 -935 -180 637 -728 557 843 168 -489 484 835 289 -454 779 -340 -797 312 128 798 37 -238 942 -781 802 714 -886 -831 167 666 -511 -984 245 -41 -39 -323 621 
869 -195 -771 137 415 -577 -814 -217 -687 235 80 116 514 723 342 -984 813 -183 691 -531 968 -353 482 -766 17 -163 -787 -600 415 -527 -147 976 267 -533 748 188 373 -509 126 -129 -232 -700 515 87 399 -825 64 -298 440 939 685 -64 334 -384 -689 -1002 -93 581 237 994 -913 768 -551 488 673 668 402 -110 -7 169 383 -599 
705 -845 -146 340 117 -668 -218 -715 -153 -418 -264 687 735 329 -351 -63 810 646 306 300 -740 -498 -934 -750 995 591 -206 -906 -224 427 -679 -122 -601 570 741 150 704 -616 682 40 -648 -855 41 160 -771 623 252 476 531 -107 7 301 -409 -336 529 -667 974 194 -492 648 570 -83 -375 -450 224 167 394 -374 -524 -856 628 575 
-207 473 248 -222 -365 113 636 913 -403 -553 252 -295 -840 -22 779 344 826 800 472 1007 -74 -115 191 371 -520 570 34 -379 640 -928 -485 -423 -974 733 150 -40 873 -499 148 703 -269 1015 634 275 458 -217 16 -197 816 -730 -371 -791 223 634 176 -442 1019 739 112 -879 180 -966 142 982 -517 403 -962 325 -949 506 -762 436 
-827 746 -169 709 195 -470 371 291 830 490 153 725 -552 508 -958 -772 -321 755 -562 -178 -831 622 554 269 590 40 387 -170 464 465 -474 1017 -392 998 219 -930 -270 122 -343 222 370 764 -43 484 -295 -389 7 581 -289 -1002 -136 34 -508 836 -844 -919 -1017 924 347 -41 -739 -693 557 321 -659 589 -537 -698 -235 945 -965 -253 
209 892 1018 72 952 -11 849 768 769 -800 -230 -217 48 -691 -782 523 -1004 423 -691 -691 -621 -480 -868 -879 -708 -338 -592 -582 544 -812 794 504 345 -252 -10 -950 667 -787 888 368 19 816 -682 7 771 -198 408 -308 587 859 -940 -539 -387 -349 878 321 4 700 -275 -475 -322 876 210 741 784 272 637 -466 -254 301 75 -883 
-572 -174 -804 -380 -271 1019 781 984 -254 566 698 -913 -750 341 -899 -404 154 770 443 555 -875 850 -596 97 697 10 -212 -692 -910 -297 133 -718 437 952 72 521 -670 -865 -498 -488 102 -853 -117 696 -727 -539 -895 406 436 -541 -668 -127 -975 -432 230 400 -51 1021 316 -890 -73 -330 207 47 279 -112 10 -346 360 2 360 -599 
435 897 913 594 546 854 586 -353 638 -557 454 -164 672 598 -809 -608 -661 -82 -467 514 -49 -82 -1019 -66 562 234 313 202 298 668 -96 -225 -166 -373 -838 757 -57 -439 605 -88 97 -966 -862 953 -1006 307 441 978 -789 -761 869 -862 -755 583 -572 474 -294 -545 940 916 1020 -393 -430 313 788 -951 -279 -62 57 227 305 177 
373 772 282 278 237 -629 660 369 -335 -1018 -695 104 -764 145 745 -462 -1005 178 -380 -775 -321 -809 218 -271 -58 575 -293 62 597 -237 -897 -305 -463 771 -442 -385 -934 608 52 -536 -484 374 620 -661 -119 -138 -338 -672 -636 92 -103 -809 765 646 962 -307 846 63 210 851 849 327 -272 -204 -494 381 989 78 492 600 312 255 
592 778 493 477 378 -362 -852 -440 -201 -896 -867 394 347 666 319 -464 -474 6 380 176 -759 924 619 -394 774 849 871 585 -221 636 -453 -920 -187 -834 -864 -966 272 -293 -168 -738 254 455 -181 360 -307 518 -470 -209 739 366 -74 255 962 237 -218 555 650 -686 760 -617 -864 248 -118 692 -399 23 428 -840 -635 696 445 866 
43 212 -368 189 788 -688 -364 -608 -788 -5 -539 -204 12 384 397 -657 268 -704 -93 -702 -89 468 -914 -607 593 754 985 135 -852 454 -324 564 -191 890 -227 -461 -711 -729 299 120 469 156 -537 314 242 282 -109 -290 -392 -21 645 -582 -817 -277 450 388 240 -199 -96 -192 784 -265 -793 292 460 976 -103 272 -489 -930 -1013 -899 
-838 -416 -138 -106 -496 37 672 388 600 831 196 -975 14 18 769 -663 -160 -171 53 48 822 232 675 -345 321 -965 -522 -392 -140 -594 737 -164 312 -280 -695 595 980 -401 266 305 418 715 -438 -822 -110 372 284 -823 -318 -988 667 848 472 849 -919 -971 122 -203 -99 -886 -939 -1012 976 -18 777 511 -324 -94 -309 -102 -1006 -350 
681 -974 32 99 -664 -302 -358 -896 -250 -280 -484 137 -332 -922 -1001 826 355 162 915 993 348 -304 631 749 -596 -291 -357 -421 728 -817 751 -234 723 781 273 759 632 198 554 -11 -639 974 546 -100 348 856 -66 24 773 559 -552 -916 -848 242 703 100 -371 530 -579 714 971 211 921 670 -120 -30 215 -37 906 -685 468 801 
566 87 54 -775 -487 493 -405 865 775 -761 -532 -797 235 474 -768 -150 288 266 -45 -529 599 -685 -462 611 725 -15 857 -79 -76 199 -799 736 843 1004 -839 -32 -501 262 777 -12 -571 34 -65 456 227 161 -640 -712 1017 773 422 -903 564 453 -3 -829 -267 -720 -729 -557 -984 886 -301 597 162 660 -566 192 -166 718 -521 -584 
901 296 -137 528 -417 647 -687 523 659 -207 -52 -809 486 126 221 -6 1022 -754 -81 990 -96 -629 848 -221 -142 735 -132 30 730 1003 -796 -701 434 389 -225 -824 -80 -46 368 -137 854 11 137 902 -1023 -233 -303 -246 -114 -720 -553 -649 121 -949 194 -105 -581 -441 840 604 872 -383 -282 741 -287 922 1021 -435 -26 825 324 97 
66 257 169 17 -914 -732 916 -929 -690 373 -752 816 788 -121 -10 -782 -173 46 -432 -364 631 -934 724 -336 -163 -529 538 -336 611 -496 960 -908 -490 82 -538 -406 -885 379 919 -28 56 -535 -605 974 301 585 695 815 -701 30 -308 -201 -235 1010 143 990 1024 -3 881 -364 -2 -895 271 214 -575 -127 860 -406 -612 595 -44 -143 
595 127 1017 857 -443 -390 -368 -217 240 525 -514 727 -803 277 -233 -20 -523 -470 772 -311 362 -2 345 -314 -543 737 217 733 383 -779 419 -868 628 -663 983 -906 518 990 -278 -88 948 337 -470 -351 -593 -899 377 -1024 -666 -74 449 169 -941 -979 72 17 -896 -897 -426 76 -696 384 978 -815 461 -753 -169 964 -855 927 -848 939 
1011 -736 677 284 323 -476 -813 410 -269 -287 879 -602 66 17 -518 -730 946 601 -853 393 371 175 -700 172 172 -42 124 -20 -782 -427 220 518 144 -545 703 -363 -684 -531 -298 -837 537 907 15 427 -35 -896 -792 -527 -533 679 550 517 215 606 -286 762 -127 -86 -674 480 807 70 43 104 293 968 -198 -452 -376 689 -483 401 
-106 -947 -805 -349 491 -218 899 -642 -282 882 832 -395 662 130 246 -674 -110 -301 -65 276 -168 -1012 -42 -272 -779 -623 391 666 643 16 -284 85 -574 -817 -476 -458 88 134 -312 207 994 -624 -903 -910 52 -932 741 907 260 795 -528 187 179 -967 888 425 -779 14 126 310 486 642 -925 -221 -761 603 -532 246 438 285 468 123 
334 -989 382 -916 508 -684 385 -516 82 -466 760 -978 913 -513 380 876 189 616 -906 380 138 -125 1012 -166 -126 -214 -857 45 359 -912 -801 -801 551 -257 -539 -852 524 625 976 388 -497 445 313 -51 -537 -452 228 -730 101 -976 147 621 -69 225 105 525 648 -740 605 968 252 966 882 801 -994 947 769 693 166 -935 712 1004 
245 791 792 146 956 -950 461 150 475 479 802 -551 577 -580 -230 -283 728 -98 594 -116 178 -103 174 -638 -430 -468 857 -746 288 431 1009 980 -5 -956 -1000 342 455 520 -785 -826 -174 -908 275 -741 -544 -136 -459 645 -603 -297 682 648 219 -130 779 -836 893 -592 40 982 -708 294 987 -712 -824 707 -456 868 -679 -746 -534 440 
-506 671 -976 820 406 -718 386 -655 -296 -903 -979 572 -486 395 -1004 606 124 -354 558 158 154 851 842 609 -397 399 454 -575 -377 -606 -875 -525 671 -444 911 1006 938 -448 1 945 -69 -765 307 457 254 583 537 -618 -72 412 -458 -336 53 -341 359 -854 -575 -672 95 -427 -126 -847 757 112 -982 -226 910 784 947 -397 -684 -720 
1014 -635 -788 -133 656 -497 230 -216 -505 -471 -163 -829 137 205 -455 -69 211 268 -1021 -789 467 -73 -118 -426 -309 91 -144 989 934 -210 509 567 225 -707 856 510 -72 116 665 753 852 868 897 -686 -276 -139 -916 1020 793 -257 -352 -394 125 300 -190 -115 812 272 736 786 77 -317 -233 -631 -160 283 34 464 -3 244 799 -878 
147 364 349 611 573 52 192 988 44 594 989 900 -248 -22 -259 -597 -499 -336 134 525 579 -357 224 -612 293 -833 769 -860 -304 470 426 -383 -328 658 -193 729 120 -778 989 -643 607 778 -244 335 403 179 481 402 542 -118 -650 -94 114 -978 890 976 -686 -332 -423 49 -225 -231 204 -803 208 -356 -547 -739 -675 -900 -377 -243 
19 -165 -442 248 -108 -933 794 -765 251 -722 181 0 101 364 -630 272 186 805 707 -561 832 375 -281 1008 -77 -668 -730 -942 748 319 311 667 -958 -328 94 -716 453 392 -931 504 -509 -401 -524 156 -914 624 519 591 -320 765 -991 -356 571 -211 -65 -840 -370 408 -548 -844 -339 -127 -363 -805 -631 -829 620 -827 682 985 532 930 
75 355 841 -271 130 -815 731 103 853 124 370 228 226 210 -609 -937 -130 422 -250 323 -485 541 -590 -124 -299 -109 -270 169 615 -238 981 778 882 -791 826 -741 577 304 -44 171 508 471 -493 446 409 -66 486 371 -952 931 40 -435 641 -678 1008 -250 -313 145 -902 971 167 -124 -41 777 -324 86 -32 -158 475 459 730 133 
237 310 -1020 507 716 881 -57 -37 695 -460 -283 299 -435 -619 -556 -198 544 -311 330 -801 -499 201 -35 843 332 -127 435 299 -405 -248 308 394 -700 -746 -790 -348 -192 652 -358 -319 506 679 875 730 381 -885 -141 -811 954 158 755 -687 -566 25 948 452 -554 630 -320 -496 401 639 -98 -419 -230 400 825 877 -53 849 -923 -481 
547 -452 181 -440 691 411 725 -869 -1006 -709 -531 -1020 -50 -705 -773 152 49 264 767 -794 -449 517 155 509 -381 -773 -809 -725 -147 -953 423 -838 -25 -925 -985 530 -100 -190 -739 84 -980 -752 265 -599 -349 -907 847 820 548 469 -743 905 839 127 -489 -616 275 128 318 -586 826 -615 -987 -62 960 -805 879 138 -197 817 -808 855 
-850 282 -582 379 -345 -715 970 -232 982 447 958 -510 104 197 534 -158 -795 20 -134 -490 -308 850 -637 -548 629 -319 620 -230 77 -904 988 1018 -455 -386 -774 754 -66 241 -501 979 -775 142 -69 -971 824 -799 -571 992 249 942 -424 -164 518 308 -592 116 841 -1 -115 896 142 82 -714 -29 -1019 369 -805 133 -793 273 90 636 
-908 -884 292 648 154 -989 9 -341 30 -947 -273 -81 720 -256 869 668 -578 293 -573 -679 -300 538 582 -834 -781 -417 -394 -167 715 518 538 606 664 614 -619 -770 -45 -813 539 328 -304 430 -247 -287 -152 -882 1000 465 975 465 -267 -100 -838 246 513 -998 703 -651 21 -870 97 -589 229 -173 806 -491 -314 -533 775 -200 786 919 
-404 -951 603 334 -817 -75 -163 -465 -724 563 87 -110 422 -386 816 -212 -546 1009 988 -63 383 336 784 -994 334 862 -829 495 491 -787 156 279 164 118 69 816 -872 573 -377 422 -654 741 -766 -31 494 272 -871 -69 -193 -7 -774 475 110 515 147 220 285 887 -136 208 -136 363 -158 65 866 563 -744 911 -37 1003 682 -237 
549 -739 -723 535 -713 618 521 -973 -229 -372 159 952 706 -180 640 403 284 298 -203 -219 86 390 -921 -186 197 346 -1 -776 248 106 34 881 -272 -381 -304 439 -630 -936 541 932 317 -155 941 -393 462 -712 -500 -800 -425 -424 -279 -644 25 -847 458 -806 714 -602 -64 -641 -742 -469 173 -388 -311 589 -652 49 -687 -329 668 32 
