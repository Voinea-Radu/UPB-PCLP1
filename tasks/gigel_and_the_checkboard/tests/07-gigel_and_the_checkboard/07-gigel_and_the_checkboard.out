24525
67 O
